cocotb/bus.py

```diff
@@ -45,41 +44,36 @@ def _build_sig_attr_dict(signals):
 
 
 class Bus(object):
-    """
-        Wraps up a collection of signals
-
-        Assumes we have a set of signals/nets named:
+    """Wraps up a collection of signals.
 
-            entity.bus_name_signal
+    Assumes we have a set of signals/nets named ``entity.<bus_name>_<signal>``.
 
-        for example a bus named "stream_in" with signals ["valid", "data"]
-            dut.stream_in_valid
-            dut.stream_in_data
+    For example a bus ``stream_in`` with signals ``valid`` and ``data`` is assumed
+    to be named ``dut.stream_in_valid`` and ``dut.stream_in_data``.
 
-        TODO:
-            Support for struct/record ports where signals are member names
+    TODO:
+        Support for struct/record ports where signals are member names.
     """
     def __init__(self, entity, name, signals, optional_signals=[], bus_separator="_", array_idx=None):
         """
         Args:
-            entity (SimHandle):   SimHandle instance to the entity containing the
-                                  bus
+            entity (SimHandle):   SimHandle instance to the entity containing the bus.
             name (str):           name of the bus. None for nameless bus, e.g.
                                   bus-signals in an interface or a modport
                                   (untested on struct/record, but could work here
-                                  as well)
+                                  as well).
```

Comment:
For these Args I think it might be appropriate to flatten them to conform to the google style docstring examples.  This is a fine fix for now.